packages/mobile/src/components/audio/AudioPlayer.tsx

```diff
@@ -353,7 +353,8 @@ export const AudioPlayer = () => {
           const nftAccessSignature = nftAccessSignatureMap[trackId]?.mp3 ?? null
           queryParams = await getQueryParams({
             audiusBackendInstance,
-            nftAccessSignature
+            nftAccessSignature,
+            userId: currentUserId
```

Comment:
Oops actually just realized we don't _need_ to do this on mobile at the moment since we don't support manager mode there.
But I'm thinking maybe we leave this in so there are no surprises if/when we introduce it and to make the request behavior consistent between platforms.
Objections?